buildbot/dependency.conf

```diff
@@ -11,23 +11,23 @@ ocl_gpu_rt_ver=20.42.18209
 ocl_gpu_rt_ver_win=27.20.100.8778
 intel_sycl_ver=build
 # https://github.com/oneapi-src/oneTBB/releases/download/v2021.1-beta08/oneapi-tbb-2021.1-beta08-lin.tgz
```

Comment:
Could you please update links accordingly?

Reply:
Hi @vladimirlaz , the latest TBB release version is https://github.com/oneapi-src/oneTBB/releases/tag/v2021.1-beta10. I prefer to update the links once the gold release version is public.  But I am Ok to update the link to beta10 now. What is your suggestion?

Reply:
We can use Beta10 version.